src/main/java/com/blamejared/compat/forestry/Still.java

```diff
@@ -0,0 +1,98 @@
+package com.blamejared.compat.forestry;
+
+import com.blamejared.ModTweaker;
+import com.blamejared.mtlib.helpers.LogHelper;
+import crafttweaker.annotations.ModOnly;
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.item.IIngredient;
+import crafttweaker.api.liquid.ILiquidStack;
+import forestry.api.recipes.IStillRecipe;
+import forestry.api.recipes.RecipeManagers;
+import forestry.factory.recipes.StillRecipe;
+import stanhebben.zenscript.annotations.Optional;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+import static com.blamejared.mtlib.helpers.InputHelper.toFluid;
+import static com.blamejared.mtlib.helpers.InputHelper.toILiquidStack;
+import static com.blamejared.mtlib.helpers.StackHelper.matches;
+
+@ZenClass("mods.forestry.Still")
+@ModOnly("forestry")
+@ZenRegister
+public class Still {
+	
+	public static final String name = "Forestry Still";
+	
+	/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+	
+	/**
+	 * Adds recipe to Still
+	 *
+	 * @param fluidOutput recipe fluid amount
+	 * @param fluidInput  recipe fluid input
+	 * @param timePerUnit time per crafting operation
+	 */
+	@ZenMethod
+	public static void addRecipe(ILiquidStack fluidOutput, ILiquidStack fluidInput, int timePerUnit) {
+		fluidOutput.amount(fluidOutput.getAmount() / 100);
```

Comment:
Magic numbers?

Reply:
It was this way in 1.10 already so I decided to keep it.
They indeed are magic numbers which seem to work.

Reply:
Okey, I tested around a bit and those numbers did exactly nothing. actually, those two lines do absolutely nothing as ILiquidStack is not mutable.
Doing it in the correct actually breaks the recipes, so I removed those two lines.